ci_tools/bumpversion.sh

```diff
@@ -0,0 +1,5 @@
+bump2version patch --message "Bump version: {current_version} to {new_version} [ci skip]"
+git config --local user.email "engineering@finbourne.com"
+git config --local user.name "finbourne-bot-public"
+git push https://slemasne:${GH_TOKEN}@github.com/slemasne/repo-test.git HEAD:$TRAVIS_BRANCH
```

Comment:
What is the reference to the repo-test here for?

Reply:
You also have the wrong username, looks like you’ve left in the test details

Reply:
Good catch. Updated in latest commit.